### ISSUE TITLE
Store genomes in links

### ISSUE THREAD

**Issue by MeAmAnUsername** [ProgrammingLife2017/Desoxyribonucleinezuur#89]
Important for #70
So for each link, store the genomes that "flow through it"

Probably the easiest way to do this:
Basic idea: for each genome, keep the previous Node it appeared in. Then go through the nodes in order and add genomes fro that node to each link.
```
Create a (hash)map with as keys genomes and as values a Node.
toposort the graph
for each node n in sorted list {
  for each genome in n {
    if genome in map and the node it maps to is a parent of n {
      add genome to link (edge) between parent and n
    }
    (re)map genome to n
  }
}
```

**Comment by MeAmAnUsername** [comment]
I have already worked on this in the branch parse-genomes (HEAD: 38f7ebab4df3ab2308f4317a90a0eb316ec202d2)

**Comment by toinehartman** [comment]
Closed by #191.
